src/CMakeLists.txt

```diff
@@ -187,6 +187,7 @@ add_object_library(clickhouse_interpreters_clusterproxy Interpreters/ClusterProx
 add_object_library(clickhouse_interpreters_jit Interpreters/JIT)
 add_object_library(clickhouse_columns Columns)
 add_object_library(clickhouse_storages Storages)
+add_object_library(clickhouse_storages_mysql Storages/MySQL)
```

Comment:
@azat how that worked before ?

Reply:
There wasn't MySQL folder before, it is added in this patch (for `MySQLSettings`)